Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Xamarin.Forms.Controls.Issues.Shared.projitems

```diff
@@ -1145,7 +1146,7 @@
     <Compile Include="$(MSBuildThisFileDirectory)Issue4356.cs">
       <DependentUpon>Issue4356.xaml</DependentUpon>
     </Compile>
-	<Compile Include="$(MSBuildThisFileDirectory)Issue4854.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)Issue4854.cs" />
```

Comment:
Any idea what is going on with this one?

Reply:
I think it just increased line number because of 26 line. It's really necessary do something about this?

Reply:
Hm seems strange, because then you would expect lines 27 through 1148 to have changed, right? As long as this class exists and things build there is no need for you to do anything about it. Was just curious why this is happening since it's not related to this change.